src/Lib/Items/CanvasImage.vala

```diff
@@ -103,6 +105,30 @@ public class Akira.Lib.Items.CanvasImage : Goo.CanvasImage, Akira.Lib.Items.Canv
         });
     }
 
+    private bool is_width_small (int width, int height) {
+        // Callculate the width if height was 1
+        var adjusted_width = width / height;
+
+        // If width is less than 1 then it is 0 we will set it to 1
+        if (adjusted_width < 1) {
+            size.width = 2;
+            return true;
+        }
```

Comment:
There's not need to store a variable if it's only used for one condition.
You can shorten this with:
```
if (width / height < 1) {
  size.width = 2;
  return true;
}
```